src/main/java/org/jenkinsci/plugins/credentialsbinding/MultiBinding.java

```diff
@@ -138,4 +141,19 @@ public Unbinder getUnbinder() {
         return (BindingDescriptor<C>) super.getDescriptor();
     }
 
+    /**
+     * Utility method for turning a collection of secret strings into a {@link Secret}.
+     * @param secrets A collection of secret strings
+     * @return A {@link Secret} generated from that collection.
+     */
+    public static Secret getSecretForStrings(Collection<String> secrets) {
```

Comment:
Best to make this `@Restricted(NoExternalUse.class)`, or move it somewhere in the `impl` package and make it package access.